openshift/templates/services/audio/pvc.yaml

```diff
@@ -0,0 +1,72 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: audio-capture-service-deploy
```

Comment:
Shared volume for audio services.  Need to figure out how to allow multiple pods to connect.